mlflow/models/docker_utils.py

```diff
@@ -37,6 +37,20 @@
 RUN pip install virtualenv
 """
 
+if os.getenv("http_proxy") is not None and os.getenv("https_proxy") is not None:
+    MAVEN_PROXY = (
+        " -DproxySet=true -Dhttp.proxyHost={http_proxy_host} "
+        "-Dhttp.proxyPort={http_proxy_port} -Dhttps.proxyHost={https_proxy_host} "
+        "-Dhttps.proxyPort={https_proxy_port} -Dhttps.nonProxyHosts=repo.maven.apache.org"
+    ).format(
+        http_proxy_host=os.environ["http_proxy"].split("//")[1].split(":")[0],
```

Comment:
Can we use [urllib.parse.urlparse](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse) instead of `split`?

Reply:
PR updated, the proxies are now parsed using the `urllib.parse.urlparse` method.